src/frontend/js/containers/user-container.jsx

```diff
@@ -0,0 +1,9 @@
+import { connect } from 'react-redux';
+import User from '../components/user';
+import { fetchUser } from '../actions/users-actions';
+
+const mapStateToProps = (state) => ({
+  user: state.get('usersReducer').get('currentUser'),
+});
+
+export default connect(mapStateToProps, { fetchUser })(User);
```

Comment:
Having access to `User` and `UserContainer` so that you can take UserContainer and add it to the render tree and then take User and write unit tests for it is great. 

One thing I don't understand is why people put connect code in a separate file. You're effectively doubling the number of components.
My first change to this pattern was to 
```
export default connect(...)(User)
export { User as UserNotConnected }
```

and you have your component and its connect code in one file. 

Works great if your app has more than 20 files